.circleci/cimodel/data/simple/ios_definitions.py

```diff
@@ -63,16 +58,16 @@ def gen_tree(self):
 WORKFLOW_DATA = [
     IOSJob(XCODE_VERSION, ArchVariant("x86_64"), is_org_member_context=False, extra_props={
         "lite_interpreter": miniutils.quote(str(int(True)))}),
-    IOSJob(XCODE_VERSION, ArchVariant("x86_64", "full_jit"), is_org_member_context=False, extra_props={
-        "lite_interpreter": miniutils.quote(str(int(False)))}),
+    # IOSJob(XCODE_VERSION, ArchVariant("x86-64", "full_jit"), is_org_member_context=False, extra_props={
```

Comment:
Just a curious question, what is the story behind `full_jit` here that it needs to be commented out?

Reply:
I commented it out/deleted it because it's not a job that's currently being run

I don't have history on why it's no longer run though